src/components/nav-bar/nav-bar.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+
+import { Link } from "gatsby";
+import * as navBarStyles from "./nav-bar.module.scss";
+
+const NavBar = () => {    
+  return (
+    <div className={navBarStyles.container}>
+      <Nav>
+        <Link to="/">Work</Link>
+        <Link to="/">Services</Link>
+        <Link to="/">Our Company</Link>
+        <Link to="/">Community</Link>
+        {` `}
+        <a href="https://xmartlabs.com">XL Website</a>
+        {` `}
+      </Nav>
+    </div>
+  );
+};
```

Comment:
```suggestion
const NavBar = () => (
    <div className={navBarStyles.container}>
      <Nav>
        <Link to="/">Work</Link>
        <Link to="/">Services</Link>
        <Link to="/">Our Company</Link>
        <Link to="/">Community</Link>
        {` `}
        <a href="https://xmartlabs.com">XL Website</a>
        {` `}
      </Nav>
    </div>
  );
```

Reply:
I changed the <Nav> because there was nothing to import it, and I typed nav, but in the HTML in the browser console it does not appear. So, are you really using it? I might delete it to see what happens.

Reply:
Yes, I got it right, the navigation bar does nothing, I delete it, nothing, I change the values, nothing.  So, I look for the component which change the Links and are nav-menu and layout.